### ISSUE TITLE
Problem running desispec tag 0.21.0

### ISSUE THREAD

**Issue by felipelm** [desihub/desispec#626]
Tried running `desispec`  tag `0.21.0` and `desiutil` tag `1.9.11` and `a674c0f9a0b302ebac234a21c2f371d46aeef18b` and got the following error:

```
2018-05-28 18:08:30,155 QLConfig WARNING : WARNING, can't open template file /app/spectro/redux/exposures/20191001/00000002/ql-mergedQA-b0-00000002.json
2018-05-28 18:08:32,873 QLConfig INFO : Checking if all the necessary files exist.
2018-05-28 18:08:32,874 QLConfig INFO : File check: Okay: /app/spectro/data/20180528/desi-00000028.fits.fz
2018-05-28 18:08:32,875 QLConfig INFO : File check: Okay: /app/spectro/data/20180528/fibermap-00000028.fits
2018-05-28 18:08:32,877 QLConfig INFO : File check: Okay: /app/spectro/redux/exposures/20180528/00000001/fiberflat-b0-00000001.fits
2018-05-28 18:08:32,879 QLConfig INFO : File check: Okay: /app/spectro/redux/exposures/20180528/00000000/psf-b0-00000000.fits
2018-05-28 18:08:32,879 QLConfig INFO : All necessary files exist for science configuration.
2018-05-28 18:08:33,261 QuickLook INFO : Reading input file /app/spectro/data/20180528/desi-00000028.fits.fz
2018-05-28 18:08:33,362 QuickLook INFO : Reading fiberMap file /app/spectro/data/20180528/fibermap-00000028.fits
2018-05-28 18:08:33,463 QuickLook INFO : Reading FiberFlat /app/spectro/redux/exposures/20180528/00000001/fiberflat-b0-00000001.fits
2018-05-28 18:08:33,669 QuickLook INFO : Finished reading all static files
2018-05-28 18:08:34,083 QuickLook ERROR : Failed to import Check_HDUs from desispec.qa.qa_quicklook. Error was 'libGL.so.1: cannot open shared object file: No such file or directory'
Traceback (most recent call last):
  File "/app/desispec/bin/desi_quicklook", line 7, in <module>
    quicklook.ql_main(quicklook.parse())
  File "/app/desispec/py/desispec/scripts/quicklook.py", line 117, in ql_main
    pipeline, convdict = quicklook.setup_pipeline(configdict)
  File "/app/desispec/py/desispec/quicklook/quicklook.py", line 555, in setup_pipeline
    if not qa.is_compatible(pa.get_output_type()):
AttributeError: 'NoneType' object has no attribute 'is_compatible'
```

Any idea of what could it be?

**Comment by Srheft** [comment]
I hadn't seen this issue before. Print the cmd. Need to know what exposure/config you are working with.

**Comment by felipelm** [comment]
desi_quicklook -i /app/desispec/py/desispec/data/quicklook/qlconfig_darksurvey.yaml -n 20180529 -c b0 -e 21 --rawdata_dir /app/spectro/data --specprod_dir /app/spectro/redux

**Comment by felipelm** [comment]
I forgot, we are mocking the exposures but this is actually exposure `4` from `20190101`.

**Comment by Srheft** [comment]
There is an example set sent to the desi-ql-qlf-int.  Let's folllow those steps. You get  desiutil issues that I do not have.

**Comment by ricardogando** [comment]
I think this is an issue with a  library that conda does not handle 
https://github.com/conda-forge/pygridgen-feedstock/issues/10 
we are looking at our libGL installation to see if we can fix this. 
On the issue above there is also a mention for matplotlib backend.
Since now you are producing plots and probably importing matplotlib, you might want to check this.
In general we use with matplotlib to avoid display issues on a cluster or remote machine without display
```
matplotlib.use('Agg')  
```

https://github.com/desihub/desispec/blob/a043f936ff0aa2845341678042b4232ce903446d/py/desispec/util.py also has an implementation

**Comment by felipelm** [comment]
Problem fixed and release on [QLF v0.3.0](https://github.com/desihub/qlf/releases/tag/v0.3.0)
